src/tests/miscellaneous/mock-adb/common-adb-configs.js

```diff
@@ -9,6 +9,49 @@ const successfulTestServerContentPath = path.join(
     '../mock-service-for-android/AccessibilityInsights',
 );
 
+const devicesCommandMatch = 'devices';
+const serviceInfoCommandMatch =
+    'shell dumpsys package com.microsoft.accessibilityinsightsforandroidservice';
+const serviceIsRunningCommandMatch = 'shell dumpsys accessibility';
+const portForwardingCommandMatch = 'forward tcp:';
+
+function addInstallServiceCommands(id, output) {
+    // These commands appear in the order that they get called
+    output[`-s ${id} shell getprop ro.build.version.sdk`] = {
+        stdout: '29',
+    };
+    output[`-s ${id} shell getprop ro.build.version.release`] = {
+        stdout: '10',
+    };
+    output[`-s ${id} help`] = {
+        stdout: '--streaming: force streaming APK directly into Package Manager',
+    };
+    output[`-s ${id} features`] = {
+        stdout:
+            'abb_exec\nfixed_push_symlink_timestamp\nabb\nstat_v2\napex\nshell_v2\nfixed_push_mkdir\ncmd',
+    };
+    output[`-s ${id} shell ls -t -1 /data/local/tmp/appium_cache 2>&1 || echo _ERROR_`] = {
```

Comment:
I didn't notice this at all when I ran locally. Is it because appium-adb didn't log it as dbug ADB, or are there commands that differ from emulator to emulator?

Reply:
It probably has to do with the version installed. This is was I was seeing with the A4 tablet